src/index.js

```diff
@@ -0,0 +1,146 @@
+const state = {
+  temp: 50,
+  city: 'Bloomington',
+  lat: 39.1670396,
+  lon: -86.5342881,
+  sky: 'sunny',
+};
+
+const upButton = document.getElementById('up-arrow');
+const downButton = document.getElementById('down-arrow');
+const tempText = document.getElementById('current-temp');
+const ground = document.getElementById('ground');
+const cityHeader = document.getElementById('city-name');
+const cityInput = document.getElementById('city');
+const realTemp = document.getElementById('realtime-temp');
+const skySelect = document.getElementById('sky-select');
+const skyLine = document.getElementById('sky');
+const landscape = document.getElementById('landscape');
+const cityReset = document.getElementById('city-reset');
+
+const setTempColor = (temp) => {
+  if (temp >= 80) {
+    tempText.style.color = 'red';
+  } else if (temp >= 70) {
+    tempText.style.color = 'orange';
+  } else if (temp >= 60) {
+    tempText.style.color = 'yellow';
+  } else if (temp >= 50) {
+    tempText.style.color = 'green';
+  } else {
+    tempText.style.color = 'teal';
+  }
+};
+
+const setGroundLayout = (temp) => {
+  if (temp >= 80) {
+    ground.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  } else if (temp >= 70) {
+    ground.textContent = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else if (temp >= 60) {
+    ground.textContent = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else {
+    ground.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  }
+};
+
+const setSkyLayout = (sky) => {
+  if (sky === 'sunny') {
+    skyLine.textContent = '☁️ ☁️ ☁️ ☀️ ☁️ ☁️';
+    landscape.style.backgroundColor = '#76e0f5';
+  } else if (sky === 'cloudy') {
+    skyLine.textContent = '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️';
+    landscape.style.backgroundColor = '#8fc9f2';
+  } else if (sky === 'rainy') {
+    skyLine.textContent = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧';
+    landscape.style.backgroundColor = '#4a73a8';
+  } else if (sky === 'snowy') {
+    skyLine.textContent = '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨';
+    landscape.style.backgroundColor = '#a5c6c7';
+  }
+};
+
+const increaseTemp = () => {
+  state['temp'] += 1;
+  setTempColor(state['temp']);
+  tempText.textContent = state['temp'];
+  setGroundLayout(state['temp']);
+};
+
+const decreaseTemp = () => {
+  state['temp'] -= 1;
+  setTempColor(state['temp']);
+  tempText.textContent = state['temp'];
+  setGroundLayout(state['temp']);
+};
+
+const changeCityHeader = () => {
+  cityHeader.textContent = cityInput.value;
+};
+
+const getCityLatAndLon = () => {
+  state['city'] = cityInput.value;
+
+  axios
+    .get('http://localhost:5000/location', {
```

Comment:
You can assign this URL to a const variable so that you don't have it sitting here as a literal string value, a properly named const variable can help readability but also prevents someone from accidentally changing the value here.